www/artists/thall/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="stylesheet.css">
+
+</head>
+<body>
+  <li class="home">  <a href= "index.html"> Home</a> </li>
+  <li class="photographs"> <a href= photograpghs.html>photographs</a> </li>
+  <li class="saskartists"> <a href= https://saskartists.ca/>Sask artists</a> </li>
+  <h1 class="Name">Don hall</h1>
```

Comment:
This claims his name is Don Hall. We don't have records for a Don Hall. More importantly, Don Hall would have a file folder of dhall, not thall.